<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="style.css">
    <title>Localisation of mRNAs to mRNP Granules</title>
</head>

<div id="navbar">
    <heading 2>LINKS:</heading>
    <a href="index.html">Home</a>
    <a href="music.html">Tantalum: <br>official releases</a>
    <a href="game.html">Trench Warfare:1916</a>
    <a href="dnd.html">DnD</a>
    <a href="dissertation.html">mRNA Localisation</a>
</div>

<body>
<div class="mucho">Localisation of mRNAs
    to mRNP Granules</div>   
    <div class="boring">This literature review was written by Tolo Akers in 2017, while studying at the University of Manchester.</div>
<div class="academic"><p>
    <B>Introduction</B></p><p>In eukaryotic cells,
    information contained in DNA is sent to the cytoplasm via
    nucleocytoplasmic transport of mRNA, so that it may be translated
    into proteins. This is essential for life, and thus is tightly
    regulated at each step from transcription to translation. Part of the
    regulation of mRNA translation requires that these mRNAs are not just
    exported to the cytoplasm in bulk, but many are localised to
    particular foci such as particular organelles. These foci also
    include cytoplasmic aggregates known as mRNP granules, some motile
    and others more static, and these serve many diverse functions. Some
    of these cytoplasmic aggregates form under specific conditions, for
    example stress or developmentally-relevant cues. For example, <I>Ash1</I>
    is localised via the <I>Ash1 </I>locasome
    to budding daughter cells in yeast (Cosma,
    2004), and under stress
    conditions in yeast, two mRNP granules form: stress granules and
    P-bodies (Erickson
    and Lykke-Andersen 2011).
    These mechanisms provide the cell an opportunity for
    post-transcriptional regulation of gene regulation, and their
    prevalence and importance may be much greater than once thought
    (Pizzinga
    and Ashe, 2014).</P>
    <P>
    <BR>
    </P>
    <P >
    <B>Generalised
    nucleocytoplasmic transport</B></P>
    <P >
    Before mRNA can be
    localised, it must first be transcribed and exported from the nucleus
    through the nuclear envelope via a nuclear pore complex (NPC). NPCs
    are large and complex channels that regulate nucleocytoplasmic
    transport through the nuclear envelope in a selective manner, while
    allowing free diffusion of small molecules and ions,
    composed of approximately 30 nucleoporins (NUPs) (Grossman et al,
    2012). To facilitate
    regulation, barrier NUPs form a mesh to prevent non-specific entry or
    exit of material; and provide low-affinity, high specificity
    interactions with transport factors. On the cytoplasm-facing side are
    8 fibrils attached to a cytoplasmic ring in the inner nuclear
    membrane extending into the cytoplasm, and a nuclear basket extending
    into the nucleus on the nuclear ring on the inner nuclear membrane.
    These both mediate specific interactions for guiding transport
    factors and their cargo through the pore (Grossman
    et al, 2012).</P>
    <P >
    Most
    nucleocytoplasmic transport requires karyopherins (importins
    and exportins), which recognise a nuclear localisation or nuclear
    export signal peptide on the cargo, and also recognise a nucleotide
    export signal on RNAs allowing them to export RNA cargoes
    (Okamura et al, 2014). Karyopherins are regulated by the small
    GTPase Ran, which exists bound to GTP in the nucleus and bound to GDP
    in the cytoplasm. This creates a RanGTP/RanGDP gradient, which is
    regulated by RanGEF (Ran-GDP-exchange factor) maintaining Ran GTPase
    in a GTP-bound state in the nucleus, and RanGAP
    (Ran-GTPase-activating protein) in the cytoplasm
    (Sloan et al 2016). Importins bind to its cargo in the
    cytoplasm and transport it through the NPC, where in the nucleus it
    releases the cargo after RanGTP binds to the importin, and is
    exported in a RanGTP-dependent manner. Exportins bind cargo in the
    nucleus and is exported once bound to RanGTP. Once in the cytoplasm,
    its cargo is released upon hydrolysis of RanGTP by RanGAP (Sloan
    et al 2016).</P>
    <P >	This
    general mechanism is true for export of most types of RNA (tRNA,
    miRNA, snRNA and rRNA), however mRNA requires a specialised mechanism
    not dependent on Ran (K&ouml;hler et al, 2007).</P>
    <P ><BR>
    </P>
    <P >
    <B>Pre-mRNP assembly</B></P>
    <P>
    During yeast mRNA
    transcription, mRNAs are co-transcriptionally folded and assembled
    into pre-mRNPs (pre- messenger RNA ribonucleoproteins) by the
    THO/TREX complex (K&ouml;hler et al, 2007). The TREX
    (TRanscription-EXport) complex is a highly conserved eukaryotic
    multi-protein complex, that functionally couples steps of mRNA
    biogenesis such as transcription, processing, decay and
    nucleocytoplasmic export. The THO subcomplex of TREX follows RNA
    polymerase II during transcription elongation, facilitating the
    interaction of other factors and folding nascent mRNA transcripts to
    assemble the pre-mRNP. Members of the THO subcomplex bind to
    chromatin, while members of TREX associate with newly-transcribed
    mRNA. TREX is continuously loaded onto nascent mRNA transcripts in
    this manner during elongation.</P>
    <P>
    In humans, the TREX
    complex is associated with splicing machinery and mRNA 5' capping in
    contrast to transcription in yeast. Human TREX is recruited by cap
    binding subunit CBP80, which interacts with TREX via the Aly/Ref
    subunits. Aly/Ref can also associate with the exon-junction complex
    deposited onto pre-mRNA during splicing.</P>
    <P >
    In yeast, the general
    export receptor for mRNPs is Mex67-Mtr2.
    This receptor is unrelated to karyopherins and functions independent
    of Ran. Like karyopherins, Mex67-Mtr2 can interact with barrier NUPs,
    allowing it to translocate through NPCs. Export adaptor Yra1 (part of
    the TREX complex) can interact with Mex67-Mtr2. However unlike other
    export receptors that are carried through to the cytoplasm, following
    recruitment of Mex67-Mtr2, Yra1 is dissociated within the nucleus via
    ATP-dependent remodelling by Sub2 (Sloan et al, 2016).</P>
    <P>
    <BR>
    </P>
    <P >
    <B>mRNA export</B></P>
    <P >
    Export
    adaptor Sac3 can interact with TREX and recruit Mex67-Mtr2 export
    receptor, and is part of a complex with Thp1, Sus1 and Cdc31 known as
    TREX-2. Sus1 is known to interact as part of another complex with
    large transcription initiation complex SAGA, that interacts with
    histones via catalysing deubiquitylation (K&ouml;hler
    et al, 2007). SAGA also
    directly contacts Mlp1 on the NPC nuclear basket, anchoring genes to
    the NPC. Sus1 is also necessary for TREX-2 to become tethered
    to an NPC via Nup1 and Nup60 on the
    nuclear basket (Garc&iacute;a-Oliver
    et al, 2012). TREX-2 therefore, via Sus1, may functionally couple
    SAGA-dependent gene expression and mRNA export.</P>
    <P>
    Sac3 also interacts with
    the Mediator complex, a regulator of RNA polymerase II (Pol II) via
    Cdk8 kinase (Schneider et al., 2015). TREX-2 therefore also may play
    a role in transcription regulation, and coupling transcription and
    mRNA export.</P>
    <P>
    In
    order to promote unidirectional movement of mRNPs, remodelling
    machinery on NPC cytoplasmic fibrils may prevent backsliding of the
    mRNP and release it from the Mex67-Mtr2 export receptor.
    ATP-dependent RNA helicase Dbp5, when associated with Nup159 on NPC
    fibrils, releases Mex67-Mtr2 in the cytoplasm (Okamura et al, 2014;
    Sloan et al, 2016). Dbp5 activity is generally low, but can be
    stimulated by mRNA export factor Gle1 and its cofactor signalling
    molecule inositol hexakisphosphate (Okamura et al, 2014).
    Directionality is in this way maintained, ratcheting the mRNP
    out into the cytoplasm.</P>
    <P>
    Npl3,
    an SR protein, may assist in both  mediating stable interaction with
    export receptor and imparting directionality for export. Npl3 in
    cytoplasm is phosphorylated by Sky1 kinase which allows its nuclear
    import. Once imported, Npl3 can associate with nascent pre-mRNA
    transcripts, and is dephosphorylated by Glc7 nuclear phosphatase.
    This facilitates Npl3 stabilising interaction with the export
    receptor, and after translocation Sky1 kinase again phosphorylates
    and releases Npl3, assisting the release of the export receptor from
    the mRNP into the cytoplasm (Sloan et al., 2016).</P>
    <P>
    <BR>
    </P>
    <P >
    <B>mRNA localisation</B></P>
    <P>
    Once exported, specific
    mRNPs may be localised via mechanisms for specific purposes. One such
    mechanism in yeast is specific to mRNPs carrying <I>ASH1</I>
    transcripts. ASH1 is necessary for repressing homothallic switching
    endonuclease (HO) expression (Sil and
    Herskowitz, 1996). HO is a site-specific endonuclease
    responsible for mating type switching, which is repressed in budding
    daughter cells while active in the mother cell. This is due to <I>ASH1</I>
    mRNA transcripts being asymmetrically localised in the tip of the
    daughter cell. The process begins in the nucleus, where localisation
    domains on <I>ASH1</I> transcripts become bound to She2p (B&ouml;hl
    et al., 2000). This process is enhanced by the non-specific
    DNA-binding protein Loc1p (Long et
    al., 2001). She2p-<I>ASH1 </I>is exported through NPCs to the
    cytoplasm where She1p-Myo4p bind, with She3p acting as an adaptor.
    This assembled mRNP, known as a locasome, is localised to the bud tip
    via Myo4p transport along actin tubules (B&ouml;hl
    et al., 2000; Long et al., 2001). While in transit, <I>ASH1
    </I>expression is repressed by accessory proteins Puf6p and Khd1p
    (Buxbaum et al., 2014).</P>
    <P >
    <I>ASH1</I>
    is not alone in being asymmetrically localised at the bud tip,
    however the localisation domain is not conserved across other
    bud-localised mRNAs,
    although most have hairpin-like secondary structure. Many however
    seem to be She2p-dependent, which suggests a more complex recognition
    process involving localisation domains, secondary and tertiary
    structure (Jambhekar
    and Derisi, 2007).</P>
    <P >
    Localisation
    of mRNA is co-translational to the endoplasmic reticulum (ER). An
    n-terminal signal peptide sequence on the emerging protein is
    recognised by a signal recognition particle (SRP) which stalls
    further translation and promotes localisation of the mRNP to the ER
    (Keenan
    et al., 2001).
    However this general mechanism is redundant; SRP-independent and
    translation-independent mechanisms mean that disruption of the SRP
    pathway does not cause serious phenotypes (Mutka
    and Walter, 2001; Kraut-Cohen et al., 2013).</P>
    <P >
    While
    mitochondria have
    their own genome, they are mostly dependent on mRNA
    and gene products from the cell nucleus (Ernster &amp; Schatz, 1981).
    There is evidence of post- and co-translational transport to meet
    this demand (Neupert, &amp; Herrmann, 2007; Gadir et al., 2011).
    Conserved mitochondria-targeting elements in mRNA have been found
    (Sylvestre, Margeot, &amp; Jacq, 2003), and deletion of which
    impair respiratory function (Margeot et al., 2002). RNA-binding
    proteins such as Puf3p and Tom20p have been shown to target mRNA to
    mitochondria (Eliyahu et al., 2010; Saint-Georges, Y. et al., 2008;
    Garc&iacute;a-Rodr&iacute;guez et al., 2007).</P>
    <P >
    Peroxisome
    biogenesis involves localised mRNAs coding for peroxins and
    peroxisome matrix proteins (Zipor
    et al., 2009), with some of this localisation requiring Puf5p
    RNA-binding protein. Some peroxisome-associated mRNAs however were
    found to localise with the ER.</P>
    <P >
    Localisations of mRNPs
    also occur in cytoplasm, in self-assembling aggregates known as mRNP
    granules (Buchan,
    2014). These exist transiently in most cases, self-
    assembling and disassembling according to cellular conditions or
    developmental state, and will be discussed further in more detail.</P>
    <P ><BR>
    </P>
    <P >
    <B>General Properties of
    mRNP Granules </B>
    </P>
    <P >
    Generally
    speaking, mRNP granules are cytoplasmic, self-assembling aggregates
    in eukaryotic cells containing non-translating mRNPs. These include
    stress granules, P-bodies, neuronal transport granules and germ
    granules (Buchan and Parker, 2009; Buchan, 2014; Jain and Parker,
    2013; Kiebler
    et al., 2006). While
    there is evidence of small mRNP aggregates in unstressed yeast cells
    that are sites of localised translation (Lui, J. et al.,
    2014), mRNP granules tend to
    contain mRNA arrested at translation initiation which do not respond
    to appropriate translation-promoting signalling (Buchan and Parker,
    2009).</P>
    <P >
    Interactions
    within and between mRNP granules are dynamic. Certain
    types of mRNA and mRNA-binding proteins may relocalise from one
    granule to another in response to changes in cellular conditions or
    time (Buchan, 2014), and granules may even dock and fuse with each
    other (Kedersha
    et al., 2005).</P>
    <P>
    Most mRNP granules cannot
    assemble without non-translating RNA. For example, P-bodies and
    stress granules cannot form when mRNAs are trapped on polysomes
    during translation using cycloheximide or emitine (Sheth and Parker,
    2003; Kedersha et al., 2000). 
    </P>
    <P>
    Granules
    assemble via proteins with self-interaction domains possibly acting
    as scaffolds. Deletion of self-interaction domains impair granule
    assembly, e.g.: PGL-3 in <I>C. elegans</I>
    P-granules (Hanazawa,
    2011), FMRP in
    <I>Drosophila</I> neuronal
    transport granules (Gareau
    et al., 2013), Edc3 in
    <I>Saccharomyces Cerevisiae</I>
    P-bodies (Decker at al., 2007) and G3BP in mammalian stress granules
    (Tourri&egrave;re
    et al., 2003). Other
    granule components also contain self-interaction motifs and
    facilitate granule assembly (e.g.: Staufen and Pat1) and also
    directly interact with other granule components (Martel
    et al., 2010; Ozgur et al.,
    2010). Proteins
    with prion-like domains contribute to granule assembly, e.g.:
    TIA-1/Pub1 in stress granules and Lsm4 in yeast P-bodies (Gilkes
    et al., 2004; Reijns et al., 2008).
    At a sufficient
    concentration such proteins containing prion-like domains assemble
    &ldquo;hydrogel&rdquo; structures in vitro, and interact with other
    prion-like domains in other mRNP granule -related proteins further
    suggesting a possible role in granule assembly (Kato
    et al., 2012).
    However, it remains to be found how these interactions remain dynamic
    and reversible in mRNP granules <I>in vivo</I>.</P>
    <P >
    Assembly and disassembly
    is assisted by the cytoskeleton and associated motor proteins, and
    this is specific to conditions and granule type. Microtubule
    depolymerising drugs inhibit large stress granules and germ granules
    in zebrafish embryos, and P-bodies become larger and less mobile
    (Ivanov et al., 2003; Theusch et al.,
    2006; Sweet et al. 2007). Specific motor proteins localise in
    stress granules to facilitate assembly and disassembly (Loschi
    et al., 2009). 
    </P>
    <P >
    Disassembly occurs when
    associated mRNAs exit to an alternative RNP granule or enter
    translation, and mRNA decay also causes mRNP granule disassembly
    (Buchan, 2014). This is most relevant to P-bodies (containing mRNA
    decay machinery and evidence of active mRNA decay (Sheth and
    Parker,2003)), however, blocking mRNA decay also increases number of
    stress granules in yeast which may be due to exchange of mRNPs
    between the two (Buchan et al., 2008). Granules other than P-bodies
    also contain mRNA-decaying enzymes, suggesting this mechanism may be
    more widespread (Buchan, 2014). Proteasomal degradation of protein
    components (e.g.: scaffold factors) may assist mRNP granule
    disassembly, and mRNP granules themselves may be degraded by
    autophagy (Buchan, 2014).</P>
    <P >
    The two most
    well-characterised mRNP granules in yeast are stress granules and
    P-bodies.</P>
    <P>
    <BR>
    </P>
    <P >
    <B>Stress Granules</B></P>
    <P>
    Stress granules form in
    eukaryotic cells in response to stressors such as starvation, heat
    shock and severe ethanol exposure (Buchan, 2014; Parton
    et al., 2014). They contain 40s ribosomal subunits and
    translation initiation factors, but the mRNAs contained are stalled
    at translation initiation (Erickson
    and Lykke-Andersen 2011; Pizzinga
    and Ashe, 2014). 
    </P>
    <P>
    The exact composition of
    stress granules are dependent on the type and duration of the
    stressor. For example, glucose starvation stress granules do not
    contain eIF3, found in heat shock- and ethanol-induced stress
    granules (Buchan and Parker, 2009). Combinations of mild stressors
    (e.g.: mild heat and ethanol stress) induce stress granules that
    otherwise do not form when exposed to one or the other (Yamamoto
    and Izawa. 2013).</P>
    <P>
    Certain mRNA-binding
    proteins are specific to stress granules, and certain mRNAs not found
    in stress granules can be found in P-bodies (Erickson
    and Lykke-Andersen 2011).</P>
    <P>
    <BR>
    </P>
    <P >
    <B>P-Bodies</B></P>
    <P>
    P-bodies are sites of
    mRNA storage and decay induced by certain stressors, conserved
    throughout eukaryotes (Erickson and
    Lykke-Andersen 2011; Buchan, 2014). Their complete composition
    is unknown, but they are known to contain mRNA decay factors (e.g.:
    Dcp1p-Dcp2p (mRNA decapping
    enzymes 1 and 2), Pat1p (topoisomerase II-associated protein), Dhh1p
    (DexD/H-box helicase 1) and Edc3p (decapping enhancer) Xrn1p (5'-3'
    ribonuclease)), translation initiation factors (e.g.: eIF4E and
    eIF3G) and Pab1p (poly(A)-binding protein) (Kulkarni
    et al., 2010; Pizzinga and Ashe, 2014).
    
    </P>
    <P>
    Inhibition
    of translation initiation and polysome dispersion favours P-body
    assembly, while trapping mRNAs on polysomes in translation
    elongation phase inhibits P-body assembly (Pizzinga
    and Ashe, 2014). Rapid cytoplasmic accumulation of
    untranslated mRNAs interact with specific RNA-binding proteins and
    mRNA decay factors to assemble P-bodies. They may also assemble from
    smaller mRNA granules that are normally translationally active, after
    polysome run-off following stress (e.g.: glucose or amino acid
    starvation) and recruiting mRNA decay factors (Lui et al., 2014). 
    </P>
    <P>
    P-bodies are dynamic.
    Different mRNAs are exchanged at different rates and some can escape
    P-bodies and re-enter translation when restarting growth phase, or
    switching to unstressed conditions (Brengues,
    2005; Bhattacharyya et al., 2006)<B>.</B> Resuming translation
    only occurs for some mRNAs, particularly those coding for ribosomal
    proteins, while others are degraded in the P-body (Arribere,
    2011).</P>
    <P>
    <BR>
    </P>
    <P >
    <B>Functions of mRNP
    Granules</B></P>
    <P >
    The
    functional relevance of mRNP granules, no matter what their specific
    type, hinges on the properties they have in common; namely specific
    mRNA localisation in response to specific stimuli. Localising
    specific mRNAs is itself a function, a well-studied example being the
    <I>ASH1</I>
    locasome during budding in yeast (Cosma, 2004). Neuronal transport
    RNPs may also function in this way (Kiebler,
    2006).</P>
    <P>
    In stress granules,
    specific mRNAs are sequestered and stalled at translation initiation
    during stress, which may function to conserve resources (Buchan,
    2014). The cell can also recover quickly from stress once conditions
    are fit and the granule disassembled, whereupon the stably stored
    mRNAs can resume translation. Germ granules, such as chromatoid
    bodies, stabilise developmentally-relevant mRNAs until a
    developmental cue triggers their exit from the granule and re-entry
    into translation (Buchan, 2014; Nguyen
    et al., 2009). The presence of mRNA stabilising proteins such
    as HuR and Pab1 in P-bodies and stress granules may be evidence of
    this function (Buchan, 2014). On the other hand, fluorescence
    recovery after bleaching (FRAPS) studies have shown that some
    components of mRNP granules, including mRNAs, exhibit shuttling
    behaviour (Buchan and Parker, 2009; Buchan et al., 2008). It may be
    the case however, that a particular fraction of mRNAs are stably
    stored simultaneous to those being rapidly shuttled (Buchan, 2014).</P>
    <P>
    P-bodies may engage in
    some mRNA storage, but likely function to facilitate efficient mRNA
    decay during stress. This is because p-bodies sequester both specific
    mRNAs and mRNA decay factors, increasing the local concentration
    which may facilitate greater rates of interaction. Further evidence
    comes from the detection of mRNA decay intermediates in the vicinity
    of P-bodies (Erickson and
    Lykke-Andersen 2011; Buchan, 2014).</P>
    <P>
    Translational repression
    may be another function of mRNP granule formation. One method being
    simply sterically restricting ribosome access, which may be true in
    some cases, but seems an unlikely mechanism for mRNP granules that
    contain both ribosomes and silenced mRNAs (Buchan, 2014). Another
    mechanism might be due to the local concentration of translational
    repressors found in mRNP granules (Kulkarni
    et al., 2010; Erickson and Lykke-Andersen 2011; Buchan, 2014;
    Pizzinga and Ashe, 2014). 
    </P>
    <P>
    By the same logic as
    localising concentrations of mRNAs and repressor proteins can promote
    repression of translation, stress granules and neuronal transport
    granules might promote translation. Both examples contain
    translational components, and therefore might promote much more rapid
    entry into translation once the granule has disassembled (Buchan and
    Parker, 2008). This function would be beneficial to the cell as
    translation can resume much quicker than otherwise, leading to much
    quicker recovery from stress. Further evidence comes from polysomes
    seen via electron microscopy on the surface of <I>Drosophila </I>polar
    granules and mammalian P-bodies (Amikura
    et al., 2001; Cougot et al., 2013).</P>
    <P>
    Interestingly, there
    exist mRNA granules in yeast that are translationally active under
    normal conditions (Lui et al., 2014). These seem to at least in part
    function to facilitate speedy assembly of stress granules and
    P-bodies, however the fact that specific mRNAs are being localised
    together suggests a possible advantage to the cell. In doing so,
    translation of functionally related proteins can be coordinated,
    which may facilitate stoichiometric production and interactions for
    efficient assembly into complexes (Lui et al., 2014).</P>
    <P >
    <BR>
    </P>
    <P >
    <B>Conclusion</B></P>
    <P >
    As more becomes known
    about the prevalence and relevance of mRNP granules, more questions
    must be asked. Firstly, it would seem sensible to assume these
    structures have a functional relevance, and this may be better
    elucidated by further understanding the specific composition and
    conditions under which these granules assemble. While there are many
    features and properties in common between types of granule, and all
    granules play a role in post-transcriptional mRNA regulation in some
    way, there are clear differences between them that indicate very
    different functions. As more becomes known about mRNP granules and
    mRNA localisation, it may be that more functionally specific subtypes
    can be found that better explain the great variability in composition
    depending on cellular conditions. It may be that mRNP granules
    perform context-specific post-transcriptional regulation on a grander
    scale than is currently understood.</P><br><br><br>
    
    <B>References</B></P>
    <P>Amikura,
    R, M Kashikawa, A Nakamura, and S Kobayashi. 2001. &ldquo;Presence of
    Mitochondria-Type Ribosomes Outside Mitochondria in Germ Plasm of
    Drosophila Embryos.&rdquo; <I>Proceedings of the National Academy of
    Sciences of the United States of America</I> 98 (16): 9133&ndash;38.
    doi:10.1073/pnas.171286998.</P>
    <P>Arribere,
    Joshua A., Jennifer A. Doudna, and Wendy V. Gilbert. 2011.
    &ldquo;Reconsidering Movement of Eukaryotic mRNAs between Polysomes
    and P Bodies.&rdquo; <I>Molecular Cell</I> 44 (5): 745&ndash;58.
    doi:10.1016/j.molcel.2011.09.019.</P>
    <P>Bhattacharyya,
    S. N., R. Habermacher, U. Martine, E. I. Closs, and W. Filipowicz.
    2006. &ldquo;Stress-Induced Reversal of microRNA Repression and mRNA
    P-Body Localization in Human Cells.&rdquo; In <I>Cold Spring Harbor
    Symposia on Quantitative Biology</I>, 71:513&ndash;21.
    doi:10.1101/sqb.2006.71.038.</P>
    <P>B&ouml;hl,
    F, C Kruse, a Frank, D Ferring, and R P Jansen. 2000. &ldquo;She2p, a
    Novel RNA-Binding Protein Tethers ASH1 mRNA to the Myo4p Myosin Motor
    via She3p.&rdquo; <I>The
    EMBO Journal</I> 19 (20):
    5514&ndash;24. doi:10.1093/emboj/19.20.5514.</P>
    <P>Brengues,
    M. 2005. &ldquo;Movement of Eukaryotic mRNAs Between Polysomes and
    Cytoplasmic Processing Bodies.&rdquo; <I>Science</I> 310 (5747):
    486&ndash;89. doi:10.1126/science.1115791.</P>
    <P>Buchan,
    J Ross. 2014. &ldquo;mRNP Granules.&rdquo; <I>RNA
    Biology</I> 11 (8): 1019&ndash;30.
    doi:10.4161/15476286.2014.972208.</P>
    <P>Buchan,
    J. Ross, and Roy Parker. 2009. &ldquo;Eukaryotic Stress Granules: The
    Ins and Outs of Translation.&rdquo; <I>Molecular Cell</I>.
    doi:10.1016/j.molcel.2009.11.020.</P>
    <P>Buchan,
    J. Ross, Denise Muhlrad, and Roy Parker. 2008. &ldquo;P Bodies
    Promote Stress Granule Assembly in Saccharomyces Cerevisiae.&rdquo;
    <I>Journal of Cell Biology</I> 183 (3): 441&ndash;55.
    doi:10.1083/jcb.200807043.</P>
    <P>Buxbaum,
    Adina R., Gal Haimovich, and Robert H. Singer. 2014. &ldquo;In the
    Right Place at the Right Time: Visualizing and Understanding mRNA
    Localization.&rdquo; <I>Nature
    Reviews Molecular Cell Biology</I>
    16 (2). Nature Publishing Group: 95&ndash;109. doi:10.1038/nrm3918.</P>
    <P>Cosma,
    Maria Pia. 2004. &ldquo;Daughter-Specific Repression of Saccharomyces
    Cerevisiae HO: Ash1 Is the Commander.&rdquo; <I>EMBO
    Reports</I> 5 (10): 953&ndash;57.
    doi:10.1038/sj.embor.7400251.</P>
    <P>Cougot,
    Nicolas, Anne-Elisabeth Molza, Emmanuel Giudice, Annie Cavalier,
    Daniel Thomas, and Reynald Gillet. 2013. &ldquo;Structural
    Organization of the Polysomes Adjacent to Mammalian Processing Bodies
    (P-Bodies).&rdquo; <I>RNA Biology</I> 10 (2): 314&ndash;20.
    doi:10.4161/rna.23342.</P>
    <P>Decker,
    Carolyn J., Daniela Teixeira, and Roy Parker. 2007. &ldquo;Edc3p and
    a Glutamine/asparagine-Rich Domain of Lsm4p Function in Processing
    Body Assembly in Saccharomyces Cerevisiae.&rdquo; <I>Journal of Cell
    Biology</I> 179 (3): 437&ndash;49. doi:10.1083/jcb.200704147.</P>
    <P>Eliyahu,
    Erez, Lilach Pnueli, Daniel Melamed, Tanja Scherrer, Andr&eacute; P
    Gerber, Ophry Pines, Doron Rapaport, and Yoav Arava. 2010. &ldquo;Tom20
    Mediates Localization of mRNAs to Mitochondria in a
    Translation-Dependent Manner.&rdquo; <I>Molecular and Cellular
    Biology</I> 30 (1): 284&ndash;94. doi:10.1128/MCB.00651-09.</P>
    <P>Erickson,
    Stacy L, and Jens Lykke-Andersen. 2011. &ldquo;Cytoplasmic mRNP
    Granules at a Glance.&rdquo; <I>Journal of Cell Science</I> 124 (Pt
    3): 293&ndash;97. doi:10.1242/jcs.072140.</P>
    <P>Ernster,
    L., and G. Schatz. 1981. &ldquo;Mitochondria: A Historical Review.&rdquo;
    <I>Journal of Cell Biology</I>. doi:10.1083/jcb.91.3.227s.</P>
    <P>Gadir,
    Noga, Liora Haim-Vilmovsky, Judith Kraut-Cohen, and Jeffrey E Gerst.
    2011. &ldquo;Localization of mRNAs Coding for Mitochondrial Proteins
    in the Yeast Saccharomyces Cerevisiae.&rdquo; <I>RNA (New York, N.Y.)</I>
    17 (8): 1551&ndash;65. doi:10.1261/rna.2621111.</P>
    <P>Garc&iacute;a-Oliver,
    Encar, Varinia Garc&iacute;a-Molinero, and Susana Rodr&iacute;guez-Navarro.
    2012. &ldquo;MRNA Export and Gene Expression: The SAGA-TREX-2
    Connection.&rdquo; <I>Biochimica et Biophysica Acta - Gene Regulatory
    Mechanisms</I> 1819 (6). Elsevier B.V.: 555&ndash;65.
    doi:10.1016/j.bbagrm.2011.11.011.</P>
    <P>Garc&iacute;a-Rodr&iacute;guez,
    Luis J., Anna Card Gay, and Liza A. Pon. 2007. &ldquo;Puf3p, a
    Pumilio Family RNA Binding Protein, Localizes to Mitochondria and
    Regulates Mitochondrial Biogenesis and Motility in Budding Yeast.&rdquo;
    <I>Journal of Cell Biology</I> 176 (2): 197&ndash;207.
    doi:10.1083/jcb.200606054.</P>
    <P>Gareau,
    Cristina, David Martel, Laetitia Coudert, Samia Mellaoui, and Rachid
    Mazroui. 2013. &ldquo;Characterization of Fragile X Mental
    Retardation Protein Granules Formation and Dynamics in Drosophila.&rdquo;
    <I>Biology Open</I> 2 (1): 68&ndash;81. doi:10.1242/bio.20123012.</P>
    <P>Gilkes,
    Natalie, Nancy Kedersha, Maranath Ayodele, Lily Shen, Georg
    Stoecklin, Laura M. Dember, and Paul Anderson. 2004. &ldquo;Stress
    Granule Assembly Is Mediated by Prion-like Aggregation of TIA-1.&rdquo;
    <I>Molecular Biology of the Cell</I> 15: 5383&ndash;98.
    doi:10.1091/mbc.E04-08-0715.</P>
    <P>Grossman,
    Einat, Ohad Medalia, and Monika Zwerger. 2012. &ldquo;Functional
    Architecture of the Nuclear Pore Complex.&rdquo; <I>Annual Review of
    Biophysics</I> 41: 557&ndash;84.
    doi:10.1146/annurev-biophys-050511-102328.</P>
    <P>Hanazawa,
    Momoyo, Masafumi Yonetani, and Asako Sugimoto. 2011. &ldquo;PGL
    Proteins Self Associate and Bind RNPs to Mediate Germ Granule
    Assembly in C. Elegans.&rdquo; <I>Journal of Cell Biology</I> 192
    (6): 929&ndash;37. doi:10.1083/jcb.201010106.</P>
    <P>Ivanov,
    Pavel A., Elena M. Chudinova, and Elena S. Nadezhdina. 2003.
    &ldquo;Disruption of Microtubules Inhibits Cytoplasmic
    Ribonucleoprotein Stress Granule Formation.&rdquo; <I>Experimental
    Cell Research</I> 290 (2): 227&ndash;33.
    doi:10.1016/S0014-4827(03)00290-8.</P>
    <P>Jain,
    Saumya, and Roy Parker. 2013. &ldquo;The Discovery and Analysis of P
    Bodies.&rdquo; <I>Advances in Experimental Medicine and Biology</I>
    768: 23&ndash;43. doi:10.1007/978-1-4614-5107-5-3.</P>
    <P>Jambhekar,
    Ashwini, and Joseph L Derisi. 2007. &ldquo;Cis-Acting Determinants of
    Asymmetric, Cytoplasmic RNA Transport.&rdquo; <I>RNA
    (New York, N.Y.)</I> 13 (5):
    625&ndash;42. doi:10.1261/rna.262607.</P>
    <P>Kato,
    Masato, Tina W. Han, Shanhai Xie, Kevin Shi, Xinlin Du, Leeju C. Wu,
    Hamid Mirzaei, et al. 2012. &ldquo;Cell-Free Formation of RNA
    Granules: Low Complexity Sequence Domains Form Dynamic Fibers within
    Hydrogels.&rdquo; <I>Cell</I> 149 (4): 753&ndash;67.
    doi:10.1016/j.cell.2012.04.017.</P>
    <P>Kedersha,
    Nancy, Georg Stoecklin, Maranatha Ayodele, Patrick Yacono, Jens
    Lykke-Andersen, Marvin J. Fitzler, Donalyn Scheuner, Randal J.
    Kaufman, David E. Golan, and Paul Anderson. 2005. &ldquo;Stress
    Granules and Processing Bodies Are Dynamically Linked Sites of mRNP
    Remodeling.&rdquo; <I>Journal of Cell Biology</I> 169 (6): 871&ndash;84.
    doi:10.1083/jcb.200502088.</P>
    <P>Kedersha,
    Nancy, Michael R. Cho, Wei Li, Patrick W. Yacono, Samantha Chen,
    Natalie Gilks, David E. Golan, and Paul Anderson. 2000. &ldquo;Dynamic
    Shuttling of TIA-1 Accompanies the Recruitment of mRNA to Mammalian
    Stress Granules.&rdquo; <I>Journal of Cell Biology</I> 151 (6):
    1257&ndash;68. doi:10.1083/jcb.151.6.1257.</P>
    <P>Keenan,
    R J, D M Freymann, R M Stroud, and P Walter. 2001. &ldquo;The Signal
    Recognition Particle.&rdquo; <I>Annual
    Review of Biochemistry</I>
    70: 755&ndash;75. doi:10.1146/annurev.biochem.70.1.755.</P>
    <P>Kiebler,
    Michael A., and Gary J. Bassell. 2006. &ldquo;Neuronal RNA Granules:
    Movers and Makers.&rdquo; <I>Neuron</I> 51 (6): 685&ndash;90.
    doi:10.1016/j.neuron.2006.08.021.</P>
    <P>Kiebler,
    Michael A., and Gary J. Bassell. 2006. &ldquo;Neuronal RNA Granules:
    Movers and Makers.&rdquo; <I>Neuron</I> 51 (6): 685&ndash;90.
    doi:10.1016/j.neuron.2006.08.021.</P>
    <P>K&ouml;hler,
    Alwin, and Ed Hurt. 2007. &ldquo;Exporting RNA from the Nucleus to
    the Cytoplasm.&rdquo; <I>Nature Reviews Molecular Cell Biology</I> 8
    (10): 761&ndash;73. doi:10.1038/nrm2255.</P>
    <P>Kraut-Cohen,
    Judith, Evgenia Afanasieva, Liora Haim-Vilmovsky, Boris Slobodin, Ido
    Yosef, Eitan Bibi, and Jeffrey E Gerst. 2013. &ldquo;Translation- and
    SRP-Independent mRNA Targeting to the Endoplasmic Reticulum in the
    Yeast Saccharomyces Cerevisiae.&rdquo; <I>Molecular
    Biology of the Cell</I> 24
    (19): 3069&ndash;84. doi:10.1091/mbc.E13-01-0038.</P>
    <P>Kulkarni,
    Meeta, Sevim Ozgur, and Georg Stoecklin. 2010. &ldquo;On Track with
    P-Bodies&rdquo; 38: 242&ndash;51. doi:10.1042/BST0380242.</P>
    <P>Long,
    Roy M., Wei Gu, Xiuhua Meng, Graydon Gonsalvez, Robert H. Singer, and
    Pascal Chartrand. 2001. &ldquo;An Exclusively Nuclear RNA-Binding
    Protein Affects Asymmetric Localization of ASH1 mRNA and Ash1p in
    Yeast.&rdquo; <I>Journal of
    Cell Biology</I> 153 (2):
    307&ndash;18. doi:10.1083/jcb.153.2.307.</P>
    <P>Loschi,
    Mariela, Claudia C Leishman, Neda Berardone, and Graciela L
    Boccaccio. 2009. &ldquo;Dynein and Kinesin Regulate Stress-Granule
    and P-Body Dynamics.&rdquo; <I>Journal of Cell Science</I> 122 (Pt
    21): 3973&ndash;82. doi:10.1242/jcs.051383.</P>
    <P>Margeot,
    Antoine, Corinne Blugeon, Julien Sylvestre, St&eacute;phane Vialette,
    Claude Jacq, and Marisol Corral-Debrinski. 2002. &ldquo;In
    Saccharomyces Cerevisiae, ATP2 mRNA Sorting to the Vicinity of
    Mitochondria Is Essential for Respiratory Function.&rdquo; <I>EMBO
    Journal</I> 21 (24): 6893&ndash;6904. doi:10.1093/emboj/cdf690.</P>
    <P>Martel,
    Catherine, Samuel Dugr&eacute;-Brisson, Karine Boulay, Billy Breton,
    Gabriel Lapointe, Sylvain Armando, V&eacute;ronique Tr&eacute;panier,
    Thomas Ducha&icirc;ne, Michel Bouvier, and Luc Desgroseillers. 2010.
    &ldquo;Multimerization of Staufen1 in Live Cells.&rdquo; <I>RNA (New
    York, N.Y.)</I> 16 (3): 585&ndash;97. doi:10.1261/rna.1664210.</P>
    <P>Mutka,
    S. C., and P. Walter. 2001. &ldquo;Multifaceted Physiological
    Response Allows Yeast to Adapt to the Loss of the Signal Recognition
    Particle-Dependent Protein-Targeting Pathway.&rdquo; <I>Molecular
    Biology of the Cell</I> 12
    (3): 577&ndash;88. doi:10.1091/mbc.12.3.577.</P>
    <P>Neupert,
    Walter, and Johannes M. Herrmann. 2007. &ldquo;Translocation of
    Proteins into Mitochondria.&rdquo; <I>Annual Review of Biochemistry</I>
    76 (1): 723&ndash;49. doi:10.1146/annurev.biochem.76.052705.163409.</P>
    <P>Nguyen
    Chi, Mai, Fr&eacute;d&eacute;ric Chalmel, Eric Agius, Nathalie Vanzo,
    Khalid S A Khabar, Bernard J&eacute;gou, and Dominique Morello. 2009.
    &ldquo;Temporally Regulated Traffic of HuR and Its Associated
    ARE-Containing mRNAs from the Chromatoid Body to Polysomes during
    Mouse Spermatogenesis.&rdquo; <I>PLoS ONE</I> 4 (3).
    doi:10.1371/journal.pone.0004900.</P>
    <P>Okamura,
    Masumi, Haruko Inose, and Seiji Masuda. 2014. &ldquo;RNA Export
    through the NPC in Eukaryotes.&rdquo; <I>Genes</I> 6 (1): 124&ndash;49.
    doi:10.3390/genes6010124.</P>
    <P>Ozgur,
    Sevim, Marina Chekulaeva, and Georg Stoecklin. 2010. &ldquo;Human
    Pat1b Connects Deadenylation with mRNA Decapping and Controls the
    Assembly of Processing Bodies.&rdquo; <I>Molecular and Cellular
    Biology</I> 30 (17): 4308&ndash;23. doi:10.1128/MCB.00429-10.</P>
    <P>Parton,
    Richard M, Alexander Davidson, Ilan Davis, and Timothy T Weil. 2014.
    &ldquo;Subcellular mRNA Localisation at a Glance.&rdquo; <I>Journal
    of Cell Science</I> 127 (Pt 10): 2127&ndash;33.
    doi:10.1242/jcs.114272.</P>
    <P>Pizzinga,
    Mariavittoria, and Mark P Ashe. 2014. &ldquo;Yeast mRNA Localization:
    Protein Asymmetry, Organelle Localization and Response to Stress.&rdquo;
    <I>Biochemical Society Transactions</I> 42 (4): 1256&ndash;60.
    doi:10.1042/BST20140086.</P>
    <P>Reijns,
    Martin A M, Ross D Alexander, Michael P Spiller, and Jean D Beggs.
    2008. &ldquo;A Role for Q/N-Rich Aggregation-Prone Regions in P-Body
    Localization.&rdquo; <I>Journal of Cell Science</I> 121 (Pt 15):
    2463&ndash;72. doi:10.1242/jcs.024976.</P>
    <P>Saint-Georges,
    Yann, Mathilde Garcia, Thierry Delaveau, Laurent Jourdren, Stephane
    Le Crom, Sophie Lemoine, Veronique Tanty, Frederic Devaux, and Claude
    Jacq. 2008. &ldquo;Yeast Mitochondrial Biogenesis: A Role for the PUF
    RNA-Binding Protein puf3p in mRNA Localization.&rdquo; <I>PLoS ONE</I>
    3 (6). doi:10.1371/journal.pone.0002293.</P>
    <P>Schneider,
    Maren, Doris Hellerschmied, Tobias Schubert, Stefan Amlacher, Vinesh
    Vinayachandran, Rohit Reja, B. Franklin Pugh, Tim Clausen, and Alwin
    K&ouml;hler. 2015. &ldquo;The Nuclear Pore-Associated TREX-2 Complex
    Employs Mediator to Regulate Gene Expression.&rdquo; <I>Cell</I> 162
    (5): 1016&ndash;28. doi:10.1016/j.cell.2015.07.059.</P>
    <P>Sheth,
    Ujwal, and Roy Parker. 2003. &ldquo;Decapping and Decay of Messenger
    RNA Occur in Cytoplasmic Processing Bodies.&rdquo; <I>Science (New
    York, N.Y.)</I> 300 (5620): 805&ndash;8. doi:10.1126/science.1082320.</P>
    <P>Sil,
    Anita, and Ira Herskowitz. 1996. &ldquo;Identification of an
    Asymmetrically Localized Determinant, Ash1p, Required
    for Lineage-Specific Transcription of the Yeast HO Gene.&rdquo; <I>Cell</I>
    84 (5): 711&ndash;22. doi:10.1016/S0092-8674(00)81049-1.</P>
    <P>Sloan,
    Katherine E., Pierre Emmanuel Gleizes, and Markus T. Bohnsack. 2016.
    &ldquo;Nucleocytoplasmic Transport of RNAs and RNA-Protein
    Complexes.&rdquo; <I>Journal of Molecular Biology</I> 428 (10).
    Elsevier B.V.: 2040&ndash;59. doi:10.1016/j.jmb.2015.09.023Review.</P>
    <P>Sweet,
    Thomas J, Brooke Boyer, Wenqian Hu, Kristian E Baker, and Jeff
    Coller. 2007. &ldquo;Microtubule Disruption Stimulates P-Body
    Formation.&rdquo; <I>RNA</I> 13 (4): 493&ndash;502.
    doi:10.1261/rna.355807.</P>
    <P>Sylvestre,
    J, A Margeot, and C Jacq. 2003. &ldquo;The Role of the 3&prime;
    Untranslated Region in mRNA Sorting to the Vicinity of Mitochondria
    Is Conserved from Yeast to Human Cells.&rdquo; <I>Molecular Biology
    of &hellip;</I> 14 (9): 3848&ndash;56. doi:10.1091/mbc.E03.</P>
    <P>Theusch,
    Elizabeth V., Kimberly J. Brown, and Francisco Pelegri. 2006.
    &ldquo;Separate Pathways of RNA Recruitment Lead to the
    Compartmentalization of the Zebrafish Germ Plasm.&rdquo;
    <I>Developmental Biology</I> 292 (1): 129&ndash;41.
    doi:10.1016/j.ydbio.2005.12.045.</P>
    <P>Tourri&egrave;re,
    H&eacute;l&egrave;ne, Karim Chebli, Latifa Zekri, Brice Courselaud,
    Jean Marie Blanchard, Edouard Bertrand, and Jamal Tazi. 2003. &ldquo;The
    RasGAP-Associated Endoribonuclease G3BP Assembles Stress Granules.&rdquo;
    <I>Journal of Cell Biology</I> 160 (6): 823&ndash;31.
    doi:10.1083/jcb.200212128.</P>
    <P>Yamamoto,
    Yosuke, and Shingo Izawa. 2013. &ldquo;Adaptive Response in Stress
    Granule Formation and Bulk Translational Repression upon a Combined
    Stress of Mild Heat Shock and Mild Ethanol Stress in Yeast.&rdquo;
    <I>Genes to Cells</I> 18 (11): 974&ndash;84. doi:10.1111/gtc.12090.</P>
    <P>
    Zipor, Gadi, Liora
    Haim-Vilmovsky, Rita Gelin-Licht, Noga Gadir, Cecile Brocard, and
    Jeffrey E Gerst. 2009. &ldquo;Localization of mRNAs Coding for
    Peroxisomal Proteins in the Yeast, Saccharomyces Cerevisiae.&rdquo;
    <I>Proceedings of the National Academy of Sciences of the United
    States of America</I> 106 (47): 19848&ndash;53.
    doi:10.1073/pnas.0910754106.</P></div>
    
</body>
</html>
